xml/Microsoft.JScript.Vsa/ResInfo.xml

```diff
@@ -54,8 +54,8 @@
         <Parameter Name="isLinked" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="resinfo">To be added.</param>
-        <param name="isLinked">To be added.</param>
+        <param name="resinfo">Specifies resource information.</param>
+        <param name="isLinked">Indicates the item is linked.</param>
         <summary>This class, member, or property is internal to the script engine and should not be called from your code.</summary>
```

Comment:
When you see this kind of boilerplate text on the APIs you're working on, make sure we add something meaningful here too and that there is a tag forInternalUseOnly on the API, like we have here in line 61.

```suggestion
        <summary>Initializes a new instance of the <see cref="T:Microsoft.JScript.Vsa.ResInfo" /> class with the specified resource information.</summary>
```